command/agent/agent_endpoint.go

```diff
@@ -222,7 +222,7 @@ func (s *HTTPServer) updateServers(resp http.ResponseWriter, req *http.Request)
 
 	// Set the servers list into the client
 	s.agent.logger.Printf("[TRACE] Adding servers %+q to the client's primary server list", servers)
-	if err := client.SetServers(servers); err != nil {
+	if _, err := client.SetServers(servers); err != nil {
```

Comment:
Not sure why SetServers changed to return the number of endpoints if its not being used here where its called

Reply:
We decided to change this so it has the same interface as `Join` for servers, as we both use them as handles to `RetryJoin`.